### DISCUSSION TITLE
clipboard.writeText not working in 1.99.9?

### DISCUSSION THREAD

**Discussion by ramiroaraujo** [johnlindquist/kit#1366] [Errors]
I've just update, it ceased to work. I tried Force Repair without success. Permissions look fine, I even tried with a simple proof of concept :( I'l downgrade in the meantime

**Comment by ramiroaraujo** [comment]
Hmm, It also doesn't work on 1.99.36? I'll try reverting a bit more

**Reply by ramiroaraujo** [reply]
Works with `1.94.19`, which is the one I had previously

**Reply by johnlindquist** [reply]
Is there any more to the script you're testing? Can you share the entire script that's not working

**Reply by johnlindquist** [reply]
Nevermind, I found the issue. Starting a new build soon.
